Assets/Scripts/Tutorial/LivingRoomTutorial.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LivingRoomTutorial : Tutorial
+{
+    public override bool Completed
+    {
+        get
+        {
+            return PlayerPrefsUtil.CompletedLivingRoomTutorial;
+        }
+    }
+
+    [SerializeField]
+    GameObject currencyPanelObject;
+    [SerializeField]
+    GameObject dogInHome;
+    [SerializeField]
+    DogCollarSlot dogCollarSlot;
+    [SerializeField]
+    GameObject dogCollarSlotObject;
+    [SerializeField]
+    GameObject dogBrowserObject;
+    [SerializeField]
+    GameObject dogHomeSlotObject;
+    [SerializeField]
+    DogHomeSlot dogHomeSlot;
+    [SerializeField]
+    GameObject shopButtonObject;
+    [SerializeField]
+    UIButton shopButton;
+    [SerializeField]
+    GameObject shelterButtonObject;
+
+    TutorialEvent defaultStartEvent = TutorialEvent.DogInHome;
+
+    protected override void setReferences()
+    {
+        base.setReferences();
+    }
+
+    protected override void fetchReferences()
+    {
+        base.fetchReferences();
+    }
+
+    protected override void subscribeEvents()
+    {
+        base.subscribeEvents();
+        dogHomeSlot.SubscribeToClickWhenOccupied(handleDogHomeSlotClick);
+        dogCollarSlot.SubscribeToClickWhenFree(handleDogCollarSlotClick);
+        shopButton.SubscribeToClick(handleShopButtonClick);
+    }
+
+    protected override void unsubscribeEvents()
+    {
+        base.unsubscribeEvents();
+        dogHomeSlot.UnsubscribeFromClickWhenOccupied(handleDogHomeSlotClick);
+        dogCollarSlot.UnsubscribeFromClickWhenFree(handleDogCollarSlotClick);
+        shopButton.UnsubscribeFromClick(handleShopButtonClick);
+    }
+
+    public override void StartTutorial()
+    {
+        base.StartTutorial();
+        if (tutorialEvents.ContainsKey(defaultStartEvent) && !tutorialEvents[defaultStartEvent])
+        {
+            // Default start
+            callOnStart(TutorialEvent.DogInHome);
+        }
+        else
+        {
+            // Restart unfinished tutorial event
+            callOnStart(currentTutorial, true);
+        }
+    }
+
+    protected override void onStart(TutorialEvent tutorialEvent)
```

Comment:
Not a fan of verbose case statements. Should be replaced with a dict of `TutorialEvent` to delegate handlers